mtcnn_pytorch/src/align_trans.py

```diff
@@ -246,10 +246,10 @@ def warp_and_crop_face(src_img,
     """
 
     if reference_pts is None:
-        if crop_size[0] == 96 and crop_size[1] == 112:
+        if crop_size is not None and crop_size[0] == 96 and crop_size[1] == 112:
             reference_pts = REFERENCE_FACIAL_POINTS
         else:
-            default_square = False
+            default_square = True
```

Comment:
Why?

Reply:
To avoid some bugs, but I've forgot the actual behaviour of that bug.